pkg/loader/compose/compose.go

```diff
@@ -317,7 +317,11 @@ func (c *Compose) LoadFile(files []string) (kobject.KomposeObject, error) {
 		serviceConfig := kobject.ServiceConfig{}
 		serviceConfig.Image = composeServiceConfig.Image
 		serviceConfig.Build = composeServiceConfig.Build.Context
-		serviceConfig.ContainerName = composeServiceConfig.ContainerName
+		newName := normalizeServiceNames(composeServiceConfig.ContainerName)
+		serviceConfig.ContainerName = newName
+		if newName != composeServiceConfig.ContainerName {
+			log.Infof("Container name in docker-compose has been changed from %q to %q", composeServiceConfig.ContainerName, newName)
```

Comment:
nit: can we change the log to `Container name in %q has been changed from %q to %q` where the first `%q` being variable `name` this shows what service we have made change in!